src/stylesheet.css

```
@@ -12,40 +12,93 @@ html {
 
 body {
 	padding: 1rem;
-	font-size: 16px;
+	font-size: 1rem;
+	font-size: 1rem;
 	line-height: 1.5;
-	--link-color: #C1D8FC;
+	--link-color: inherit;
 }
 
 a {
 	color: var(--link-color);
-	text-decoration: none;
+	text-decoration: underline;
+	text-underline-offset: 0.25rem;
+	text-decoration-thickness: .5px;
+	transition: all 300ms ease-in-out;
+	text-decoration: underline;
+	text-underline-offset: 0.25rem;
+	text-decoration-thickness: .5px;
+	transition: all 300ms ease-in-out;
```

did you mean to duplicate these?

```suggestion
	text-decoration: underline;
	text-underline-offset: 0.25rem;
	text-decoration-thickness: .5px;
	transition: all 300ms ease-in-out;
```